connector/mock_connector/connector.go

```diff
@@ -39,15 +39,15 @@ func (m *MockConnector) EXPECT() *MockConnectorMockRecorder {
 // Close mocks base method.
 func (m *MockConnector) Close() error {
```

Comment:
Could you re-generate this? Close method should no longer be here at all...

actually, i think nobody is even using this mock anymore. Maybe just delete the whole `mock_connector` package and `package.go` (which generates it)?

Reply:
Done